website/docs/d/policy_set_definition.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "Policy"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_policy_set_definition"
+description: |-
+  Get information about a Policy Set Definition.
+---
+
+# Data Source: azurerm_policy_set_definition
+
+Use this data source to access information about an existing Policy Set Definition.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_policy_set_definition" "example" {
+  display_name = "Policy Set Definition Example"
+}
+
+output "id" {
+  value = data.azurerm_policy_set_definition.example.id
+}
+```
+
+## Argument Reference
+
+* `name` - Specifies the name of the Policy Set Definition. Conflicts with `display_name`.
+
+* `display_name` - Specifies the display name of the Policy Set Definition. Conflicts with `name`.
+
+~> **NOTE** Since `display_name` does not have uniqueness on Azure, therefore when using `display_name` to retrieve existing policy set definitions, only one of the policy set definitions with same `display_name` will be retrieved. 
+
+* `management_group_id` - (Optional) Only retrieve Policy Set Definitions from this Management Group.
+
+## Attributes Reference
+
+* `id` - The ID of the Policy Set Definition.
+
+* `description` - The Description of the Policy Set Definition.
+
+* `policy_type` - The Type of the Policy Set Definition. Possible values are `BuiltIn` and `Custom`.
```

Comment:
We don't need to include possible types as this is a data source
```suggestion
* `policy_type` - The Type of the Policy Set Definition.
```

Reply:
Done